### ISSUE TITLE
Doesn't report home away correctly

### ISSUE THREAD

**Issue by garrettgalow** [lforrest/isthereagiantsgametoday#29]
I believe the issue is on this line: https://github.com/lforrest/isthereagiantsgametoday/blob/master/js/site.js#L58

```
if (todaysGame.location == "AT&T Park") {
                $("body").addClass("home");
                $("#yesno .homeaway").text("At home");
}
```

Effectively it is parsing home games as "AT&T Park" for the location when I think it now includes " - San Francisco"

**Comment by tmack8001** [comment]
Should just do a substring search in case it changes again, or something more clever. Thanks for the great service that helps all the times planning my commutes.

**Comment by garrettgalow** [comment]
Fixed in PR #28
